src/org/omegat/util/gui/LookAndFeelHelper.java

```diff
@@ -0,0 +1,113 @@
+     *     and re-licensed into GPL3.
+     * @return true when dark theme, otherwise false.
+     */
+    public static boolean isDarkTheme() {
+        if(isDark != null) {
+            return isDark;
+        }
+        // Based on tests with different LAFs and color combinations, a light
+        // theme can be reliably detected by observing the brightness value of
+        // the HSB Values of Table.background and Table.foreground
+        //
+        // Results from the test (Theme / Foreground / Background)
+        // Gtk - Numix (light) / 0.2 / 0.97
+        // Gtk - BlackMATE (dark) / 1.0 / 0.24
+        // Gtk - Clearlooks (light) / 0.1 / 1.0
+        // Gtk - ContrastHighInverse (dark) / 1.0 / 0.0
+        // Gtk - DustSand (light) / 0.19 / 1.0
+        // Gtk - TraditionalOkTest (light) / 0.0 / 0.74
+        // Gtk - Menta (light) / 0.17 / 0.96
+        // DarkNimbus (dark) / 0.9 / 0.19
+        // DarkMetal (dark) / 0.87 / 0.19
+        // CDE (light) / 0.0 / 0.76
+        // Nimbus (light) / 0.0 / 1.0
+        // Metall (light) / 0.2 / 1.0
+        // Windows (light) / 0.0 / 1.0
+        // Windows Classic (light) / 0.0 / 1.0
+        // Windows HighContrast Black (dark) / 1.0 / 0
+        Color foreground = UIManager.getColor("Table.foreground");
+        Color background = UIManager.getColor("Table.background");
+        float foreground_brightness = Color.RGBtoHSB(
+                foreground.getRed(),
+                foreground.getGreen(),
+                foreground.getBlue(),
+                null)[2];
+        float background_brightness = Color.RGBtoHSB(
+                background.getRed(),
+                background.getGreen(),
+                background.getBlue(),
+                null)[2];
+        isDark = background_brightness < foreground_brightness;
+        return isDark;
+    }
+
+    /**
+     * Set LookAndFeel default and load application colors
+     */
+    public static void setDefaultLaf() {
+        // Workaround for JDK bug 6389282 (OmegaT bug bug 1555809)
+        // it should be called before setLookAndFeel() for GTK LookandFeel
+        // Contributed by Masaki Katakai (SF: katakai)
+        UIManager.getInstalledLookAndFeels();
+
+        try {
+            UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());
+            if (isDarkTheme()) {
+                UIManager.put("OmegaT.AlternatingHilite",
+                        UIManager.getColor("TextArea.background").brighter());  // NOI18N
+            } else {
+                UIManager.put("OmegaT.AlternatingHilite",
+                        UIManager.getColor("TextArea.background").darker());  // NOI18N
+            }
+            UIManager.put("OmegaT.SpecialForeground", Color.BLACK);
+            UIManager.put("OmegaT.SpecialBackground", new Color(0xC8DDF2));
+        } catch (Exception e) {
+            // do nothing
+            Log.logErrorRB("MAIN_ERROR_CANT_INIT_OSLF");
```

Comment:
We won't need the string for this in `Bundle.properties` anymore either.

Reply:
removed